ibis-server/docs/development.md

```diff
@@ -42,7 +42,6 @@ To run the tests:
 
 ### Environment Variables
 - `WREN_ENGINE_ENDPOINT`: The endpoint of the Wren Java engine
-- `LOG_LEVEL`: The log level of the server (default is INFO)
```

Comment:
Curiously, it seems that `loguru` doesn't use this env. How can we change the log level after moving to `loguru`?

Reply:
The main principle of `loguru` is that there are no levels. By default, everything is printed to `sys.stderr`. It then allows you to choose to output specific levels to designated files. Please refer to https://github.com/Delgan/loguru/issues/138#issuecomment-525844687

Reply:
So, everything is logged to `sys.stderr`. IMO, `sys.stderr` is not always printed in the console. Maybe we should check whether the environment in the docker container will print it or not.

Reply:
After testing, the log will be printed in the console.